styleguide/components/translated/_example-cy.html.haml

```diff
@@ -0,0 +1,2 @@
+- I18n.locale = :cy
```

Comment:
This is the one part I'd like to find a better approach for. Because we load haml via a webpack loader, there's not a great way to dynamically _change_ language outside of this inline Ruby hack. Would mean lots of little partials to show welsh variants in storybook. 💭 Thoughts welcome

Reply:
What about altering it when the user changes their language? And setting the locale then for the user based on the language they've switched to.

Reply:
Because of the way we load haml files in storybook there's not an obvious _place_ to do that. But yeah, that'd be the ideal.